config/routes.rb

```diff
@@ -38,6 +39,12 @@
   resources :incurred_incidentals
   resources :reservations
 
+  #resources :rentals do
+  #  resources :incurred_incidentals, except: [:destroy] do
+  #    patch :active, on: :member
+  #  end
+  #end
```

Comment:
I was hopeful that my original way would one day be deemed superior and wanted to keep it, just in case...  Will remove.